tequilapi/endpoints/list_identities_test.go

```diff
@@ -0,0 +1,38 @@
+package endpoints
+
+import (
+	"net/http/httptest"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/julienschmidt/httprouter"
+	"github.com/mysterium/node/identity"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestListIdentities(t *testing.T) {
+
+	req := httptest.NewRequest("GET", "/irrelevant", nil)
+	resp := httptest.NewRecorder()
+
+	mockIdm := newManager("0x000000000000000000000000000000000000000A")
+	handlerFunc := ListIdentitiesEndpointFactory(mockIdm).ListIdentities
+	handlerFunc(resp, req, httprouter.Params{})
+
+	assert.JSONEq(
+		t,
+		`{
+            "identities":["0x000000000000000000000000000000000000000A"]
```

Comment:
Lets have identity as structure, because it will have more fields later:
```
{
    "id": "0x000000000000000000000000000000000000000A"
}
```

Reply:
done